google-cloud-secret_manager/samples/snippets.rb

```diff
@@ -430,6 +430,40 @@ def update_secret project_id:, secret_id:
   secret
 end
 
+def update_secret_with_alias project_id:, secret_id:
+  # [START secretmanager_update_secret_with_alias]
+  # project_id = "YOUR-GOOGLE-CLOUD-PROJECT"  # (e.g. "my-project")
+  # secret_id  = "YOUR-SECRET-ID"             # (e.g. "my-secret")
+
+  # Require the Secret Manager client library.
+  require "google/cloud/secret_manager"
+
+  # Create a Secret Manager client.
+  client = Google::Cloud::SecretManager.secret_manager_service
+
+  # Build the resource name of the secret.
+  name = client.secret_path project: project_id, secret: secret_id
+
+  # Create the secret.
+  secret = client.update_secret(
+    secret: {
+      name: name,
+      versionAliases: {
```

Comment:
The test is failing, I think because this is supposed to be in snake case (`version_aliases`)